soc/xtensa/intel_adsp/Kconfig

```diff
@@ -47,13 +47,25 @@ config ADSP_WIN0_OFFSET
 	  "firmware status register" at boot) within the reserved
 	  region at the start of HP-SRAM.
 
+config ADSP_WIN2_OFFSET
```

Comment:
Note that I have something WIP to move all of this to devicetree, right now memory window configuration is done in multiple places (kconfig, headers, dts, etc.), so this all will go away and we have just one place. so, this can go first as is and I will adapt my patches...

Reply:
@nashif See my comment in https://github.com/zephyrproject-rtos/zephyr/pull/49077/files#r951296982 . I don't think these should move to DT as these are not HW properties, but FW build time properties and we have examples where the offsets are different for different products using the same hw.

Reply:
you can have whatever offsets you need in the application or FW using a DTS overlay that matches your usage.

Reply:
@nashif  Ok, if that is allowed, then these can indeed be moved over. I was does pedantically following https://docs.zephyrproject.org/latest/build/dts/dt-vs-kconfig.html and thought DTS should be reserved for HW properies, i.e. "In other words, devicetree mainly deals with hardware, and Kconfig with software.". I guess this depends on the viewpoint as well.. the value is determined by SW protocol, but it affects the SRAM layout, so maybe define in DTS still follows the highlevel guidance. This change is for another PR though, we need to clean the existing other users from SOF first.

Reply:
Ack - having the partitioning of all the windows in DT make sense to me, the cherry on the cake would be if we could feed the same DT source to the Linux SOF driver for window partitioning in the future.

Reply:
@lgirdwood The driver gets these values by querying the FW at boot. I don't think we need to hardcode these in driver.